src/libs/Notification/LocalNotification/BrowserNotifications.ts

```diff
@@ -104,7 +105,7 @@ export default {
             reportID: report.reportID,
         };
 
-        push(title, body, icon, data, onClick);
+        push(title, body, icon, data, onClick, true);
```

Comment:
Coming from here https://github.com/Expensify/App/issues/53700 Passing `silent: true` here has also disabled the custom notification sound for the web and desktop. We fixed it here https://github.com/Expensify/App/issues/53700#issuecomment-2537853394 by passing `silent: false`, and when `silent: false`, the custom sound plays.